java/api/src/main/java/io/ray/api/PlacementGroups.java

```diff
@@ -38,17 +38,7 @@ public static PlacementGroup getPlacementGroup(PlacementGroupId id) {
    * @return The placement group.
    */
   public static PlacementGroup getPlacementGroup(String name) {
-    return Ray.internal().getPlacementGroup(name, false);
-  }
-
-  /**
-   * Get a placement group by placement group name from all jobs.
-   *
-   * @param name The placement group name.
-   * @return The placement group.
-   */
-  public static PlacementGroup getGlobalPlacementGroup(String name) {
-    return Ray.internal().getPlacementGroup(name, true);
+    return Ray.internal().getPlacementGroup(name);
```

Comment:
we also need a `getPlacementGroup(name, namespace)` API, right?

Reply:
Yes. But we can add it in next PR since this PR is already a breakage.